localization/hi/observer/Readme.md

```diff
@@ -0,0 +1,156 @@
+
+|        **Title**       | **Category**                | **Language** | **Tag**                                     |
+|:----------------------:|-----------------------------|--------------|---------------------------------------------|
+| Observer (अवलोकक पैटर्न) | Behavioral (व्यावहारिक पैटर्न) | हिन्दी        | Gang Of Four(गैंग ऑफ फोर)<br/>Reactive (रीएक्टिव) |
```

Comment:
This metadata part needs to be in English. See for an example here: https://raw.githubusercontent.com/iluwatar/java-design-patterns/master/localization/es/abstract-document/README.md